src/accessors-geog.cpp

```diff
@@ -63,70 +47,66 @@ double perimeter(PyObjectGeography a, double radius = numeric_constants::EARTH_R
 }
 
 void init_accessors(py::module& m) {
+    py::options options;
+    options.disable_function_signatures();
```

Comment:
It is not possible to set this globally for all modules that will be added?

Reply:
Yes probably better to set this once in `spherely.cpp` (the `options` lifetime would go through all `init_xxx` calls). I initially thought this would be set only for some functions or classes but I ended-up setting it for all functions.